### PR TITLE
feat: upgrade to Chromium 72.0.3626.52

### PR DESCRIPTION
**Pull request by nornagon**
#### Description of Change
Ref https://github.com/electron/electron/projects/16

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: upgrade to Chromium 72.0.3626.52

### CODE REVIEW

File: atom/browser/web_view_guest_delegate.cc [hunk 1]
```diff
@@ -38,11 +38,12 @@ void WebViewGuestDelegate::AttachToIframe(
...
             content::WebContents::FromRenderFrameHost(embedder_frame));
 
+  content::WebContents* guest_web_contents = api_web_contents_->web_contents();
   // Attach this inner WebContents |guest_web_contents| to the outer
   // WebContents |embedder_web_contents|. The outer WebContents's
   // frame |embedder_frame| hosts the inner WebContents.
-  api_web_contents_->web_contents()->AttachToOuterWebContentsFrame(
-      base::WrapUnique<content::WebContents>(embedder_web_contents_),
+  guest_web_contents->AttachToOuterWebContentsFrame(
+      base::WrapUnique<content::WebContents>(guest_web_contents),
```
Comment (nornagon) [inline atom/browser/web_view_guest_delegate.cc]: this looks like it's changing behaviour, is that intentional?

File: atom/browser/api/atom_api_web_contents_impl.cc [hunk 2]
```diff
@@ -16,6 +19,13 @@ namespace atom {
 
 namespace api {
 
+void WebContents::DetachFromOuterFrame() {
+  // See detach_webview_frame.patch on how to detach.
+  auto* impl = static_cast<content::WebContentsImpl*>(web_contents());
+  impl->GetRenderManagerForTesting()->RemoveOuterDelegateFrame();
```
Comment (deepak1556) [inline atom/browser/api/atom_api_web_contents_impl.cc]: Probably a better way to access the frame host manager would be via `FrameTreeNode` https://cs.chromium.org/chromium/src/content/browser/frame_host/frame_tree_node.h?l=106 . Based on the comments `GetRenderManagerForTesting` might be removed soon.

File: patches/common/swiftshader/fix_arm64.patch [hunk 3]
```diff
@@ -0,0 +1,38 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Cheng Zhao <zcbenz@gmail.com>
+Date: Thu Dec 6 02:00:06 2018 +0000
+
+Backport fed67899a7b44c9409e94442eaca0deab2f809e3 to fix build on arm64.
+https://swiftshader.googlesource.com/SwiftShader.git/+/fed67899a7b44c9409e94442eaca0deab2f809e3%5E%21/
+
+Can be removed after updating to Chrome 72.
```
Comment (nornagon) [inline patches/common/swiftshader/fix_arm64.patch]: Do you mean 73?

Comment (zcbenz) [reply patches/common/swiftshader/fix_arm64.patch]: Yeah 😄

**Pull request merged**